data/plugins/projectsearch.lua

```diff
@@ -171,18 +172,11 @@ function ResultsView:draw()
   -- status
   local ox, oy = self:get_content_offset()
   local x, y = ox + style.padding.x, oy + style.padding.y
-  local files_number = core.project_files_number()
-  local per = common.clamp(files_number and self.last_file_idx / files_number or 1, 0, 1)
+  local per = common.clamp(1, 0, 1)
```

Comment:
Right; this is the compromise of running project search like this with this PR; since we no longer manage the whole tree (which can, and usually is tens of thousands of files), we don't know how many files there are. So this is just always 1. I'll remove the line.